src/systems/joint_controller/JointController.cc

```diff
@@ -85,6 +95,39 @@ void JointController::Configure(const Entity &_entity,
     return;
   }
 
+  if (_sdf->HasElement("use_force_commands") &&
+      _sdf->Get<bool>("use_force_commands"))
+  {
+    this->dataPtr->useForceCommands =
+      _sdf->Get<unsigned int>("use_force_commands");
```

Comment:
Shouldn't this be `bool`?

Reply:
Actually, we don't even need to parse it again. c022cb8